doc/api-extensions.md

```diff
@@ -2281,3 +2281,6 @@ When enabled and a suitable system is in use (requires ZFS 2.2 or higher), the Z
 
 This introduces support for the `all-projects` query parameter for the GET API calls to both `/1.0/operations` and `/1.0/operations?recursion=1`.
 This parameter allows bypassing the project name filter.
+
+## `metadata_configuration`
+Adds `GET /1.0/metadata/configuration` API endpoint to retrieve the generated metadata configuration in a JSON format. You can check the list of [`configuration options`](config-options.md) to learn more about the structure.
```

Comment:
Why would you mark `configuration options` as code? My suggestion should work as is.
I also suggested a "the" in the first sentence.

Reply:
Also, there's no information about the structure in that link.

Reply:
> Why would you mark `configuration options` as code? My suggestion should work as is. I also suggested a "the" in the first sentence.

Interesting - the link works fine in incremental builds, but not when building from scratch.
This should do it: 

```
{doc}`configuration options </config-options>`
```